src/components/views/messages/TextualBody.tsx

```diff
@@ -238,18 +238,19 @@ export default class TextualBody extends React.Component<IBodyProps, IState> {
     }
 
     private highlightCode(code: HTMLElement): void {
-        if (code.textContent.length > MAX_HIGHLIGHT_LENGTH) {
+        const codeTextContent = code.textContent ?? "";
```

Comment:
I assume this change is to make the strict mode checker pass? If we are checking, we may as well just early exit if the text content is empty: we can't syntax highlight the empty string.